packages/breadcrumb/src/js/breadcrumb.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/**
+ * Breadcrumb module
+ * @file Handles interaction behaviour for the breadcrumb.
+ */
+
+class Breadcrumb {
+  constructor(el) {
+    this.init(el);
+  }
+
+  init(el) {
+    const breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
+
+    // truncate if more than 4 options
+    if (breadcrumbs.length > 4) {
+      var crumbs = Array.from(breadcrumbs);
+
+      // create the expand crumb link
+      var expandCrumb = document.createElement("li");
+      expandCrumb.classList.add("uq-breadcrumb__item", "uq-breadcrumb__expand");
+      expandCrumb.innerHTML = '<a class="uq-breadcrumb__link">...</a>';
+
+      // add expand crumb after the home crumb
+      var firstCrumb = el.querySelector(".uq-breadcrumb__item:first-of-type");
```

Comment:
For the breadcrumbs we queried for the `.uq-breadcrumb__item` selector earlier, couldn't we access the first one via `this.crumbs[0]` being the first array object? This should be the home link and would save doing another query.